api-datos/cmd/tablacruce/modelos.go

```diff
@@ -33,11 +38,15 @@ type CountTableRow struct {
 	Codigo        string
 }
 
-// NewIsland Crea una nueva estructura de tipo Island
-func NewIsland(id int, nombre string, longitud, latitud float64) Island {
-	return Island{id, nombre, longitud, latitud}
-}
-
 func NewCountTableRow(nombreEspecie string, nombreIsla string, maximoNidos int, temporada int, latitud, longitud float64, codigo string) CountTableRow {
 	return CountTableRow{nombreEspecie, nombreIsla, maximoNidos, temporada, latitud, longitud, codigo}
 }
+
+type HistoricTableRow struct {
+	Temporada   int
+	MaximoNidos int
+}
+
+func NewHistoricTableRow(temporada, maximoNidos int) HistoricTableRow {
```

Comment:
Parece que falta agregar el tipo del argumento `temporada`:
```suggestion
func NewHistoricTableRow(temporada int, maximoNidos int) HistoricTableRow {
```